tests/AppBundle/Command/AddUserCommandTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tests\Command;
+
+use AppBundle\Command\AddUserCommand;
+use AppBundle\Entity\User;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class AddUserCommandTest extends KernelTestCase
+{
+    private function executeCommand(array $inputArgs, array $interactiveInputs = [])
+    {
+        self::bootKernel();
+
+        $command = new AddUserCommand();
+        $command->setApplication(new Application(self::$kernel));
+
+        $commandTester = new CommandTester($command);
+        $commandTester->setInputs($interactiveInputs);
+        $commandTester->execute(array_merge(['command' => $command->getName()], $inputArgs));
```

Comment:
Is the `command` argument required? I think we removed this in some past Symfony version ... but I could be perfectly wrong 😄